ocs_ci/helpers/helpers.py

```
@@ -7005,16 +7005,28 @@ def find_cephblockpoolradosnamespace(storageclient_uid=None):
         cephbpradosns = list(
-            filter(lambda x: f"-{storageconsumer}" in x, cephblockpool_rns_names)
+            filter(
+                lambda x: f"replicated-metadata-pool-{storageconsumer}" in x
+                or f"ocs-storagecluster-cephblockpool-{storageconsumer}" in x,
+                cephblockpool_rns_names,
+            )
         )[0]
     logger.info(
-        f"StorageClient is {storageclient_name} with uid {storageclient_uid}. StorageConsumer is {storageconsumer}"
+        f"StorageClient is {storageclient_name} with uid {storageclient_uid}. "
+        f"StorageConsumer is {storageconsumer}. Cephblockpoolradosnamespace is {cephbpradosns}."
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)